Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -181,6 +181,8 @@ def __init__(self, host, test_paths, options):
         self._potential_test_resource_files = []
 
         self.import_list = []
+        self.upstream_revision = self.test_downloader().upstream_revision
+
```

Comment:
This will always be `None` at this point, no? When the downloader hasn't done anything yet, the `upstream_revision` is just `None`, which makes creating the downloader redundant here.

It's also probably more work than we really want in the initialiser anyway? Setting `self.upstream_revision = None` here would be fine.

Also you've added an extra (redundant) new line here.

Reply:
ooops. I set it twice and I forgot to remove it. 🤦 Thanks for catching that.

Reply:
done.